lib/ansible/modules/system/xfs_quota.py

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Emmanouil Kampitakis <info@kampitakis.de>
+# Copyright: (c) 2018, William Leemans <willie@elaba.net>
+
+# Contributors:
+# (c) 2018, Emmanouil Kampitakis <info@kampitakis.de>
+# (c) 2018, Nicolas Magliaro <nicolasmagliaro@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: xfs_quota
+short_description: Set Quotas on XFS filesystems
+description:
+  - Configure Quotas on XFS filesystems. /etc/projects and /etc/projid needs to be configured before calling this module when setting project quotas.
+version_added: "2.8"
+author: "William Leemans (@bushvin) <willie@elaba.net>"
+options:
+  type:
+    description:
+      - The xfs quota type.
```

Comment:
Please add a parameter type to every parameter, so that the documentation includes this information.
```suggestion
      - The xfs quota type.
    type: str
```

Reply:
done